src/wp-admin/includes/class-wp-site-health.php

```diff
@@ -716,6 +716,41 @@ public function get_test_theme_version() {
 		return $result;
 	}
 
+	/**
+	 * Tests if the revision is on/off.
+	 *
+	 * @since 6.7.0
+	 *
+	 * @return array The revision results.
+	 */
+	public function get_test_revisions_status() {
+		// Check if revisions are disabled.
+		$revisions_disabled = defined( 'WP_POST_REVISIONS' ) && WP_POST_REVISIONS === false;
+
+		// Prepare the result array for Site Health.
+		$result = array(
+			'label'       => __( 'Post Revisions' ),
+			'status'      => $revisions_disabled ? 'recommended' : 'good',
+			'badge'       => array(
+				'label' => __( 'Content Management' ),
+				'color' => $revisions_disabled ? 'red' : 'green',
+			),
+			'description' => $revisions_disabled
+				? __( 'Post revisions are currently disabled on your site. It is recommended to enable them to track changes and recover previous versions of your content.' )
+				: __( 'Post revisions are enabled on your site. This helps with version control and content management.' ),
```

Comment:
I would actually show the revision count here as well.

Knowing that there are plugins out in the wild, that provide different review counts for different Post types, I think it would be reasonable to either try to show this automatically, or at least provide a filter and encourage the use of this for plugin authors.